vignettes/join-keys.Rmd

```diff
@@ -49,20 +108,26 @@ data_1 <- data.frame(X = factor(1:10), Y = 21:30, Z = 1:10)
 data_2 <- data.frame(W = factor(10:1), V = factor(5:14), M = rep(1:5, 2))
 
 data <- teal_data(
-  dataset("D1", data_1, code = "D1 <- data.frame(X = factor(1:10), Y = 21:30, Z = 1:10)"),
-  dataset("D2", data_2, code = "D2 <- data.frame(W = factor(10:1), V = factor(5:14), M = rep(1:5, 2))"),
+  D1 = data_1,
+  D2 = data_2,
   join_keys = join_keys(
     join_key("D1", keys = c("X")),
     join_key("D2", keys = c("V", "W"))
-    # join_key("D2", "D2", keys = c("V", "W")), # equivalent to using primary_key
+    ## equivalent to using primary_key
+    # join_key("D2", "D2", keys = c("V", "W")),
```

Comment:
```suggestion
    # join_key("D2", "D2", keys = c("V", "W")), # nolint
```

Reply:
Thanks, added a silent no lint to avoid having that code in the vignette

![image](https://github.com/insightsengineering/teal.data/assets/211358/7e4aad8c-4bf9-4875-b656-f68a870e6b67)

Reply:
Even better!